continuous_integration/install_with_flexiblas.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -e
+
+pushd ..
+ABS_PATH=$(pwd)
+popd
+
+# create conda env
+conda update -n base conda conda-libmamba-solver -q --yes
+conda config --set solver libmamba
+conda create -n $VIRTUALENV -q --yes -c conda-forge python=$PYTHON_VERSION \
+    pip cython blis mkl openblas meson-python pkg-config
+source activate $VIRTUALENV
+
+pushd ..
+
+# build & install FlexiBLAS
+mkdir flexiblas_install
+git clone https://github.com/mpimd-csc/flexiblas.git
+pushd flexiblas
+
+mkdir build
+pushd build
+
+cmake ../ -DCMAKE_INSTALL_PREFIX=$ABS_PATH"/flexiblas_install" \
+    -DEXTRA="OPENBLAS_CONDA;BLIS_CONDA;MKL_CONDA" \
+    -DFLEXIBLAS_DEFAULT=BLIS_CONDA \
+    -DOPENBLAS_CONDA_LIBRARY=$CONDA_PREFIX"/lib/libopenblas.so" \
+    -DBLIS_CONDA_LIBRARY=$CONDA_PREFIX"/lib/libblis.so" \
+    -DMKL_CONDA_LIBRARY=$CONDA_PREFIX"/lib/libmkl_rt.so"
+make
+make install
+
+# Check that all 3 BLAS are listed in FlexiBLAS configuration
+$ABS_PATH/flexiblas_install/bin/flexiblas list
+popd
+popd
+
+# build & install numpy
+git clone https://github.com/numpy/numpy.git
+pushd numpy
+git submodule update --init
+
+echo "libdir=$ABS_PATH/flexiblas_install/lib/
+includedir=$ABS_PATH/flexiblas_install/include/flexiblas/
+version=3.3.1
+extralib=-lm -lpthread -lgfortran
+Name: flexiblas
+Description: FlexiBLAS - a BLAS wrapper
+Version: \${version}
+Libs: -L\${libdir} -lflexiblas
+Libs.private: \${extralib}
+Cflags: -I\${includedir}" > flexiblas.pc
+
+PKG_CONFIG_PATH=$ABS_PATH/numpy/ pip install . -v --no-build-isolation -Csetup-args=-Dblas=flexiblas
```

Comment:
I tried it locally and it seems to work. Let's apply this on the PR to check the updated CI log.

```suggestion
PKG_CONFIG_PATH=$ABS_PATH/numpy/ pip install . -v --no-build-isolation -Csetup-args=-Dblas=flexiblas -Csetup-args=-Dlapack=flexiblas
```